### PR TITLE
Handles all the available events from event grid.

### PR DESCRIPTION
**Pull request by Ayaz43**
Update to handle all the events from all the Azure resources.

### CODE REVIEW

File: fappdeploy/httptrigger/GHubEventDispatch.cs [hunk 1]
```diff
@@ -1,130 +1,213 @@
...
+        }
+
+        var queryParams = System.Web.HttpUtility.ParseQueryString(req.RequestUri.Query);
+        string repo_name = queryParams.Get("repoName");
+        log.LogInformation("fetching repo name from query parameters." + repo_name);
+
+        if (event_data.Length > 2 && !string.IsNullOrEmpty(repo_name))
         {
+            event_type = event_source.ToLower()  ;
+
```
Comment (ashishonce) [inline fappdeploy/httptrigger/GHubEventDispatch.cs:118]: what exactly this part is doing?

File: fappdeploy/httptrigger/GHubEventDispatch.cs [hunk 2]
```diff
@@ -1,130 +1,213 @@
...
+            }
+            var req_data = requestObject[0]["data"];
+
+            switch (event_source)
+            {
+                case "AppConfiguration":
+                    req_data = ParseAppConfigurationEvent(requestObject);
+                    break;
+
+                case "Web/sites":
```
Comment (ashishonce) [inline fappdeploy/httptrigger/GHubEventDispatch.cs:131]: do we have this / in the events?

**Review by ashishonce — APPROVED** [review]

File: fappdeploy/httptrigger/GHubEventDispatch.cs (hunk 2 above)
Comment (Ayaz43) [reply fappdeploy/httptrigger/GHubEventDispatch.cs:131]: Yes, it's there and I kept it as it is.

File: fappdeploy/httptrigger/GHubEventDispatch.cs (hunk 1 above)
Comment (Ayaz43) [reply fappdeploy/httptrigger/GHubEventDispatch.cs:118]: if event triggered is Microsoft.KeyVault.KeyExpired, then
event_source will be KeyVault and 
event_type will be keyvault-keyexpired.
basically we are appending the event_source as prefix in the event_type.

**Pull request merged**